pkg/http_endpoint/v1/router.go

```diff
@@ -56,26 +60,645 @@ func authRouter(clients *common.ClientContainer,
 	return router
 }
 
-func adminRouter(authMiddleware func(http.Handler) http.Handler, secret string) *chi.Mux {
-	router := chi.NewRouter()
-	router.Use(authMiddleware)
-	router.Use(httpAuth.AdminAuthenticationMiddleware(secret))
-	/*
-		router.Get("/", func(w http.ResponseWriter, r *http.Request) {
-			_, err := w.Write([]byte("You are admin\n"))
-			if err != nil {
+// GetDatasource method gets the datasource list
+func GetDatasource(clients *common.ClientContainer) http.HandlerFunc {
```

Comment:
I think it makes sense to move all this functions to separate file, because it makes router codes unreadable. Please, refer to https://github.com/zaletniy/visualization-api/pull/17/files.